test/main/store/migrations/index.test.js

```diff
@@ -931,3 +931,55 @@ describe('migration 25', () => {
     })
   })
 })
+
+describe('migration 26', () => {
+  beforeEach(() => {
+    state = {
+      main: {
+        _version: 20,
```

Comment:
this should be 25 or else it will run a bunch of migrations that might influence your tests